app/controllers/clearance/sessions_controller.rb

```diff
@@ -33,4 +37,8 @@ def url_after_create
   def url_after_destroy
     sign_in_url
   end
+
+  def redirect_url
```

Comment:
If this is going to be a separate method, it should probably be named consistently with the other controller methods available for override [here](https://github.com/thoughtbot/clearance#overriding-controllers).

But I think maybe we should just use `url_after_create` just as if their sign in had already been successful?